lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -245,10 +247,10 @@ def add_three_d_secure(post, payment_method, options)
         'capture' => 'capture'
       }
 
-      def commit(action, params)
+      def commit(action, params, options)
```

Comment:
Since `refund` is the only method that needs to pass any options, what if you made this method definition have a default? 
```suggestion
      def commit(action, params, options = {})
```

Reply:
(And then you could remove the additional `options` arguments from the other transaction methods)